src/_posts/hands-on-ii.md

```diff
@@ -0,0 +1,177 @@
+
+<!--more-->
+
+I’ve implemented this series using a VIVE. Remember that for hand-based interactions, **a cardboard or Daydream is not enough**. You will need an advanced head-mounted display supporting room-scale VR and [6DoF](https://en.wikipedia.org/wiki/Six_degrees_of_freedom) controllers such as those included in the [HTC VIVE](https://www.vive.com/us/product/#controller-intro) or the [Oculus Touch](https://www.oculus.com/rift).
+
+All articles in the series:
+
+  1. [Hands-on! Building interactive training with A-Frame](/blog/hands-on-i/)
+  2. Hands-on! Simple physical behaviours in A-Frame
+  3. [Hands-on! Hand-based interactions in A-Frame](/blog/hands-on-iii/)
+
+## The project repository
+
+If you’ve not done so already, clone the [repository of the simulation](https://github.com/delapuente/aframe-simulation-demo). Enter the repository and run `npm install` and `npm start` to install the dependencies and start the development server at port `8080`.
+
+Look at [`step2/index.html`](https://github.com/delapuente/aframe-interactive-training/blob/master/step2/index.html) and [`js/step2`](https://github.com/delapuente/aframe-interactive-training/tree/master/js/step2) to browse the finished HTML and JavaScript code of this step. Open a browser and enter the following URL `localhost:8080` to find the index of steps. Append `step2` at the end of the URL path to play the scene after step 2 is completed (or select step 2 in the list).
+
+## The holding problem
+
+An accurate correspondence between virtual and real objects is vital to increase the sense of presence. The prblem with the hands models in the scene is that they are not oriented in the same way than your real hands hold the controllers.
+
+Take a look at the following video, focus on how the real fingertips and the virtual hand's don't match. The grab axis, around which the fingers close, is also misaligned.
+
+<video controls src="/images/blog/hands-on/step2-the-holding-problem.mp4"></video>
+
+Is it better to replace the hand controllers with the VIVE 3D models. Furthermore, in this particular simulation you will hold the tools in a similar way you naturally hold the VIVE controllers, so the accuracy of simulation is even higher if using the VIVE models. Using the `vive-controls` instead of `hand-controls` attribute also [allow us to change the model](/docs/0.6.0/components/vive-controls.html#value_model) in the future so we can provide a better representation of our hands.
```

Comment:
Alternative: `The simulation is more effective when using the 3D models of the Vive controllers since the tools you see in VR will match the real tool hold in your hand.`